backend/src/routes/rubricRouter.ts

```diff
@@ -1,305 +1,171 @@
 // Router for all /rubrics requests
 import express, { Request, Response } from "express";
-import { PrismaClient } from "@prisma/client";
-import {
-  body,
```

Comment:
factoring out the validation to its own module is great! rubricRouter was definitely getting monolithic